docs/content/alias.md

```diff
@@ -81,7 +81,7 @@ q) Quit config
 e/n/d/r/c/s/q> q
 ```
 
-Once configured you can then use `rclone` like this,
+Once configured you can then use `rclone` like this, (replace `remote` with the name you gave your remote)
```

Comment:
Good idea, thank you!

I'm thinking... I find the trailing comma of the original text a bit strange, and with your added parenthesis this is perhaps even more true. Not a big deal, of course, but what about something like the following instead?

```suggestion
Once configured you can then use `rclone` like this (replace `remote` with the name you gave your remote):
```

Reply:
Yeah that sounds better, I'll do that. Also I've only used rsync on a OneDrive remote so far but I think this should work for all of these files. It might be even more clear to say "drive" for OneDrive, google drive, etc. and different terminology for others.